solutions/prob6.py

```diff
@@ -0,0 +1,34 @@
+# https://projecteuler.net/problem=6
+# Sum square difference of first 100 natural numbers
+
+import math
+
+class sumSquareDiff(object):
+	
+	def __init__(self, number):
+        	self.number = number
+
+	def square(self, no):
+        	return no*no
+
+	def sum(self):
+		sum1=0
+		N = self.number
```

Comment:
`endrange` is constant here. You do not need to assign the `endrange` to a variable. 
We should keep memory management in mind when writing code :-).